R/commonMachineLearningClassification.R

```diff
@@ -413,7 +432,22 @@
                         distribution = "multinomial", n.cores=1) #multiple cores breaks modules in JASP, see: INTERNAL-jasp#372
       probabilities <- gbm::predict.gbm(bfit, newdata = grid, n.trees = classificationResult[["noOfTrees"]], type = "response")
       preds <- colnames(probabilities)[apply(probabilities, 1, which.max)]
-    }
+    } else if(type == "neuralnet"){
+	    hidden <- .getNeuralNetworkNodes(options)
```

Comment:
I'm not sure what hidden is here. I think it somehow refers to the hidden layers. Does it contain information about the number of hidden layers? The nodes within all hidden layers? The function .getNeuralNetworkNodes suggests that it's about the nodes, but in general it seems to refer to the network structure. Perhaps networkStructure makes more sense.